specs/StartInBackgroundMode.md

```diff
@@ -0,0 +1,101 @@
+CreationPriority
+===
+
+# Background
+WebView2 has a functionality enabled by default which raises the launch priority of WV2
+processes to ensure that WV2 processes are given higher CPU priority and access to resources 
+while creation, and that can bring improvements on time spent creating a WV2 instance. Since 
+this improvement is at the cost of taking higher CPU priority, we introduce `CreationPriority` 
+property to give developers an option to decide by themselves.
+
+# Description
+* Setting `CreationPriority` to `COREWEBVIEW2_CREATION_PRIORITY_NORMAL` will signal WebView2 
+to use less system resources during the creation. 
+* Setting `CreationPriority` to `COREWEBVIEW2_CREATION_PRIORITY_HIGH` will attempt to enable 
+the launch time improvement functionality with raising the launch priority during the creation.  
+* Default value is `COREWEBVIEW2_CREATION_PRIORITY_HIGH`.
+## Notes
+* The host app's priority is at least normal before applying the launch time improvement functionality. 
+Else, WebView2 still get created with normal priority even with `COREWEBVIEW2_CREATION_PRIORITY_HIGH` 
+setting.
+* Currently launch time improvement functionality is only enabled for WebView2 manager process, so the 

```

Comment:
We don't document the launch time improvement feature and we don't need to talk about it in our reference documentation. We can just document the CreationPriority feature directly.

```suggestion
* Currently
```